components/LayoutTransformControl/src/LayoutTransformControl.cs

```diff
@@ -0,0 +1,468 @@
+    /// matrix.
+    /// </summary>
+    private void ProcessTransform()
+    {
+        // Get the transform matrix and apply it
+        _transformation = GetTransformMatrix(Transform).Round(DecimalsAfterRound);
+
+        if (_matrixTransform != null)
+        {
+            _matrixTransform.Matrix = _transformation;
+        }
+
+        // New transform means re-layout is necessary
+        InvalidateMeasure();
+    }
+
+    /// <summary>
+    /// Walks the Transform and returns the corresponding matrix.
+    /// </summary>
+    /// <param name="transform">The transform to create a matrix for.
+    /// </param>
+    /// <returns>The matrix calculated from the transform.</returns>
+    private Matrix GetTransformMatrix(Transform transform)
+    {
+        if (transform != null)
+        {
+            // WPF equivalent of this entire method (why oh why only WPF...):
+            // return transform.Value;
+
+            // Process the TransformGroup
+            var transformGroup = transform as TransformGroup;
+
+            if (transformGroup != null)
+            {
+#if WINUI3
+                var groupMatrix = Microsoft.UI.Xaml.Media.Matrix.Identity;
+#elif WINUI2
+                var groupMatrix = Windows.UI.Xaml.Media.Matrix.Identity;
+#else
+                var groupMatrix = Matrix.Identity;
+#endif
+
+                foreach (var child in transformGroup.Children)
+                {
+                    groupMatrix = groupMatrix.Multiply(GetTransformMatrix(child));
+                }
+
+                return groupMatrix;
+            }
+
+            // Process the RotateTransform
+            var rotateTransform = transform as RotateTransform;
+
+            if (rotateTransform != null)
+            {
+                return rotateTransform.GetMatrix();
+            }
+
+            // Process the ScaleTransform
+            var scaleTransform = transform as ScaleTransform;
+
+            if (scaleTransform != null)
+            {
+                return scaleTransform.GetMatrix();
+            }
+
+            // Process the SkewTransform
+            var skewTransform = transform as SkewTransform;
+
+            if (skewTransform != null)
+            {
+                return skewTransform.GetMatrix();
+            }
+
+            // Process the MatrixTransform
+            var matrixTransform = transform as MatrixTransform;
+            if (matrixTransform != null)
+            {
+                return matrixTransform.Matrix;
+            }
+
+            if (transform is CompositeTransform)
+            {
+                throw new NotSupportedException("CompositeTransforms are not supported (yet) by the LayoutTransformControl.");
+            }
+
+            // TranslateTransform has no effect in LayoutTransform
+        }
+
+        // Fall back to no-op transformation
+#if HAS_UNO
+        return Windows.UI.Xaml.Media.Matrix.Identity;
+#else
+        return Matrix.Identity;
+#endif
```

Comment:
@Arlodotexe needed here too
```
D:\a\Windows\Windows\components\LayoutTransformControl\src\LayoutTransformControl.cs(216,16): error CS0234: The type or namespace name 'Media' does not exist in the namespace 'Windows.UI.Xaml' (are you missing an assembly reference?) [D:\a\Windows\Windows\components\LayoutTransformControl\src\CommunityToolkit.WinUI.Controls.LayoutTransformControl.csproj]
```
Think above thing you did resolved the other message, so seems like a valid solution?